A couple of parallel_sequential tests with watcher have kernels that are too large on WH

Local testing for https://github.com/tenstorrent/tt-metal/pull/42472 showed

FAILED tests/ttnn/unit_tests/operations/fused/parallel_sequential/test_parallel_sequential.py::TestShardedExecution::test_sharded_three_phase - RuntimeError: TT_FATAL @ /localdev/bbradel/tt-metal/tt_metal/impl/program/program.cpp:2215: state.offset <= max_size
info:
Program size (71792) too large for kernel config buffer (70656) on TENSIX


FAILED tests/ttnn/unit_tests/operations/fused/parallel_sequential/test_parallel_sequential.py::TestMatmulFusion::test_matmul_followed_by_n_rms[num_rms=4] - RuntimeError: TT_FATAL @ /localdev/bbradel/tt-metal/tt_metal/impl/program/program.cpp:2215: state.offset <= max_size
info:
Program size (70784) too large for kernel config buffer (70656) on TENSIX

These need to be skipped with watcher on.

There is no benefit to trying to make them fit. Therefore just the skips will be enough.

--- a/tests/ttnn/unit_tests/operations/fused/parallel_sequential/test_parallel_sequential.py
+++ b/tests/ttnn/unit_tests/operations/fused/parallel_sequential/test_parallel_sequential.py
@@ -25,7 +25,7 @@ import torch
 import ttnn
 
 
-from models.common.utility_functions import comp_pcc, skip_with_llk_assert
+from models.common.utility_functions import comp_pcc, skip_with_llk_assert, skip_with_watcher
 
 
 def stress_test_program_cache(fn):
@@ -448,6 +448,7 @@ class TestShardedExecution:
         golden = rms_norm_golden(sh_ln_golden(torch_input, weight=torch_w), torch_w)
         check_pcc(golden, out, label=f"sharded {shard_type}")
 
+    @skip_with_watcher("Program too large for kernel config buffer. Will not fix.")
     @skip_with_llk_assert("Compiler error with LLK asserts enabled. Issue: #40330")
     @stress_test_program_cache
     def test_sharded_three_phase(self, device):
@@ -711,6 +712,7 @@ class TestMatmulFusion:
         golden = torch_rms_norm(torch_rms_norm(torch_input.float(), torch_w.float()) @ torch_b.float(), torch_w.float())
         check_pcc(golden, out, label="multicore RMS->MM->RMS")
 
+    @skip_with_watcher("Program too large for kernel config buffer. Will not fix.")
     @skip_with_llk_assert("Compiler error with LLK asserts enabled. Issue #40330")
     @pytest.mark.parametrize("num_rms", [2, 3, 4])
     @stress_test_program_cache

